src/parser/utils.ts

```diff
@@ -37,16 +49,14 @@ export function isUint8Array(value: unknown): value is Uint8Array {
   return Object.prototype.toString.call(value) === '[object Uint8Array]';
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+declare const global: any;
+
 /** Call to check if your environment has `Buffer` */
 export function haveBuffer(): boolean {
```

Comment:
do we actually need to check for `haveBuffer` anymore? Because we depend on `buffer` its always there, either polyfilled or not.

Reply:
Whoosh 🌬️ and just like that its gone, good catch!